tests/test_validation.py

```diff
@@ -38,22 +39,17 @@ def setUp(self):
             rawschema = f.read()
         self.schema = yaml.safe_load(rawschema)
         self.validator = salobj.DefaultingValidator(schema=self.schema)
-        self.default = dict(host="192.168.223.14",
-                            port=17310,
-                            connection_timeout=10,
-                            read_timeout=10,
-                            )
+        self.default = dict(
+            host="192.168.223.14", port=17310, connection_timeout=10, read_timeout=10
```

Comment:
do you care that this isn't the real IP?
main crio box is 192.168.1.45, the one on the rotating part is 192.168.1.46

Reply:
Yes I do care. I would prefer to have a correct value here. The CSC only talks to the main low level controller so I assume that's the "main crio box".